plugin/editorconfig.vim

```diff
@@ -503,18 +507,20 @@ function! s:ApplyConfig(config) " {{{1
 
 " Set the indentation style according to the config values
 
-    if has_key(a:config, "indent_style")
+    if index(g:EditorConfig_disable_rules, 'indent_style') < 0 &&
+                \ has_key(a:config, "indent_style")
```

Comment:
The first one would be better. In addition, it is better to have the function be `s:IsRuleActive`, not `isRuleActive".